src/metpy/calc/basic.py

```diff
@@ -1036,6 +1037,68 @@ def smooth_n_point(scalar_grid, n=5, passes=1):
     return smooth_window(scalar_grid, window=weights, passes=passes, normalize_weights=False)
 
 
+@exporter.export
+def xarray_zoom(input_field, zoom, output=None, order=3, mode='constant', cval=0.0,
```

Comment:
I feel like this would be slightly more consistent with the rest of the API (like `isentropic_interpolation_as_dataset`) if it was `zoom_xarray`.